extensions/schema-compare/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "schema-compare",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
Do you want to change ADS version dependency too or this will work with both new and old?

Reply:
the schema compare change should still work on older ADS versions